calrissian/main.py

```diff
@@ -39,6 +39,7 @@ def activate_logging(level):
 def add_arguments(parser):
     parser.add_argument('--max-ram', type=str, help='Maximum amount of RAM to use, e.g 1048576, 512Mi or 2G. Follows k8s resource conventions')
     parser.add_argument('--max-cores', type=str, help='Maximum number of CPU cores to use')
+    parser.add_argument('--max-gpus', type=str, nargs='?', help='Maximum number of GPU cores to use')
```

Comment:
Ah, you can't infer that from the Kubernetes cluster itself?

Reply:
this is not the definiton of the limit of the cluster but the limit of the client itself in order to avoid a unwanted usage of resource (e.g. autoscaling) https://github.com/Duke-GCB/calrissian#scalability--resource-requirements